src/components/Indexer/processor.ts

```diff
@@ -118,8 +201,11 @@ class BaseEventProcessor {
   }
 
   protected checkDdoHash(decryptedDocument: any, documentHashFromContract: any): boolean {
+    INDEXER_LOGGER.logMessage(`decrypted document: ${JSON.stringify(decryptedDocument)}`)
     const utf8Bytes = toUtf8Bytes(JSON.stringify(decryptedDocument))
     const expectedMetadata = hexlify(utf8Bytes)
+    INDEXER_LOGGER.logMessage(`hash: ${create256Hash(expectedMetadata.toString())}`)
+    INDEXER_LOGGER.logMessage(`ctr hash: ${documentHashFromContract}`)
```

Comment:
What's a ctr hash? I doubt anyone will know when they see this popping up in the logs, so this is essentially just clutter. If you feel that it is important, then explain it: 

```suggestion
    INDEXER_LOGGER.logMessage(`Document Hash From Contract: ${documentHashFromContract}`)
```

Reply:
These will be deleted, should we keep for debug?

Reply:
deleted